pkg/horizon/deployment.go

```diff
@@ -158,6 +175,9 @@ func Deployment(
 			},
 		},
 	}
+	deployment.Spec.Template.Spec.Volumes = append(GetVolumes(
+                instance.Name,
+		instance.Spec.ExtraMounts), GetLogVolume())
```

Comment:
The suggested change to `GetLogVolume()` means that this would require merging the two slices at this point though.

The other way you could do it, is keep the function the same and then just append this to the `volumeMounts` variable defined on line 99 and just have everything using the same mounts.

I would probably opt for the first option though, just to keep the mounts minimal on the log pod. It just means you'll need to merge the slices here, rather than appending since this would now give you a slice of slices.

Reply:
Actually, sorry, just realised this is the Volume not the Volume mount. The only thing missing here is the `HorizonPropagation` variable. So it should be:

```suggestion
				instance.Spec.ExtraMounts, HorizonPropagation), GetLogVolume())

```

Reply:
The code in this area has changed enough to make it something to revisit (if necessary) after this patch is merged.